src/components/ship.jsx

```diff
@@ -35,6 +35,10 @@ export default React.createClass({
     return `ship-${this.state.ship.id}`;
   },
 
+  getResetStyles() {
+    return this.state.shipSettings && this.state.shipSettings.appearance && this.state.shipSettings.reset_styles;
```

Comment:
`this.state.shipSettings.reset_styles` -> `this.state.shipSettings.appearance.reset_styles` ?